imkar/scattering/analytical.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+import pyfar as pf
+
+import numbers
+
+
+def rectangular(frequencies, phis, width, length, height, c=343):
+    """
+    This function computes the scattering coefficient of periodic rectangular
+    profiles, assuming that the extent is infinite.
+
+    This function is based on Ducourneau's method described by Prof. Embrechts
+    [#]_.
+
+    Parameters
+    ----------
+    frequencies : array, double
+        Vector of frequencies the scattering coefficient is computed at.
+    phis : array, double
+        Vector of incidences phi the scattering coefficient is computed at.
+        Values between 0°-90° allowed.
+    width : double
+        Width of the rectangulars in m.
+    length : double
```

Comment:
```suggestion
    gap_width : double
```
not sure if the i renamed them correctly, but these suggesteion should be the input of the function.

Reply:
Changed, length is now defined as gap_width+rectangular_width and gap_width is the new input parameter.